docs/guides/setup.md

```diff
@@ -25,9 +25,7 @@ brew install nexus-cli
 
 #### Arch Linux
 
-The [nexus-cli](https://aur.archlinux.org/packages/nexus-cli) is also available
-in the AUR (Arch User Repository). You can install it using your preferred
-[AUR helper](https://wiki.archlinux.org/title/AUR_helpers):
+The [nexus-cli](https://aur.archlinux.org/packages/nexus-cli) is also available in the AUR (Arch User Repository). You can install it using your preferred [AUR helper](https://wiki.archlinux.org/title/AUR_helpers):
```

Comment:
We should setup a linter later, but it's recommended to [keep a maximum line length of 80 characters for markdown files](https://google.github.io/styleguide/docguide/style.html#character-line-limit) since not everyone works with word wrap enabled in their editors.

Reply:
I agree with @tmsdkeys here, I think a paragraph should be on a single line in `.md` files. If we cut it to 80 characters, the rendering can become awkward because we're unlikely to use a monospace font in the docs. Also resizing the screen will mess up the width also.

Reply:
https://www.perplexity.ai/search/when-using-gitbook-for-docs-sh-BSwmKsjPTx2Ou.YrBcETCg#0

Should be take care of by Gitbook

Reply:
@aelesbao it does look off though in gitbook: https://docs.talus.network/getting-started/setup#access-devnet-sui-explorer or https://docs.talus.network/getting-started/setup#arch-linux

Reply:
Fair enough, in that case I don't have a strong opinion here. ctrl+shift+p "word wrap" is imo easy enough to trigger in vscode. But if we have a linter that formats automatically I don't mind either way.

I'm all for linting for consistency in the docs but something post-devnet.

Reply:
> Fair enough, in that case I don't have a strong opinion here. ctrl+shift+p "word wrap" is imo easy enough to trigger in vscode. But if we have a linter that formats automatically I don't mind either way.

Scrape this. No newlines then.